docs/release-notes/highlights-1.9.1.asciidoc

```diff
@@ -18,5 +18,6 @@ This release introduces a preemptive measure to mitigate link:https://github.com
 [id="{p}-191-known-issues"]
 === Known issues
 
+- When using the Red Hat certified version of the operator automatic upgrades from previous versions of ECK do not work. To upgrade remove the old ECK operator installation and install the new version manually. Because CRDs remain in place after removal this operation should not negatively affect existing Elastic Stack deployments managed by ECK.
```

Comment:
```suggestion
- When using the Red Hat certified version of the operator, automatic upgrades from previous versions of ECK do not work. To upgrade, delete the old ECK operator installation and install the new version manually. Because CRDs remain in place after deletion, this operation should not negatively affect existing Elastic Stack deployments managed by ECK.
```

Reply:
TIL https://apps.labor.ny.gov/ux/doc/v1/design-remove-vs-delete.html. which makes me think that delete is more appropriate that remove here.

Reply:
How about uninstall? I think that is actually what it says in the OCP UI.